classifier/src/Lib.hs

```diff
@@ -363,68 +369,67 @@ inspectAttachments dataLayer ticketInfo attachments = do
 
     lastAttach      <- handleMaybe . safeHead . reverse . sort $ attachments
     att             <- handleMaybe =<< getAttachment lastAttach
-    inspectAttachment config ticketInfo att
+    inspectAttachment config ticketInfo att extractLogsFromZip extractIssuesFromLogs
   where
     handleMaybe :: Maybe a -> App a
     handleMaybe Nothing  = throwM $ AttachmentNotFound (tiId ticketInfo)
     handleMaybe (Just a) = return a
 
+-- This is to enable test cases on 'inspectAttachment'
+-- Perhaps we can add this to 'Config' ?
+type ExtractLogFileFunc = Int -> LByteString -> Either ZipFileExceptions [LogFile]
```

Comment:
This is nice as a first approximation. In the long run, it would be better to put these functions into a "layer" as we did so far since we (usually) pass these functions together in a record because they are easy to think about and organize code like that.

Reply:
I agree. Should I make a new layer and add it to `Config` ?